avocado/utils/vmimage.py

```diff
@@ -111,16 +111,15 @@ def _feed_html_parser(self, url, parser):
     def get_best_version(self, versions):
         return max(versions)
 
-    def get_version(self):
+    def get_versions(self):
         """
-        Probes the higher version available for the current parameters.
+        Return all available versions for the current parameters
```

Comment:
For this docstring, can we try to follow the PEP-0257, please? This could break some tools.

Reply:
> For this docstring, can we try to follow the PEP-0257, please? This could break some tools.

I wonder if we can have some tool to check that and avoid future regressions.